lib/index.js

```diff
@@ -44,13 +44,19 @@ const validateJson = jsonInput => {
   return { value };
 };
 
-const jsonToSchema = ({ baseType = "AutogeneratedMainType", jsonInput }) => {
+const jsonToSchema = ({
+  baseType,
+  prefix,
```

Comment:
I think providing default values makes sense here, especially since this can be used as a library as well as a cli.
```suggestion
  baseType = "AutogeneratedMainType",
  prefix = "",
```